src/modules/initcpiocfg/main.py

```diff
@@ -198,6 +202,9 @@ def find_initcpio_features(partitions, root_mount_point):
     if uses_lvm2:
         hooks.append("lvm2")
 
+    if uses_zfs:
+        hooks.append("zfs")
```

Comment:
Since there's several of these, maybe a more-Pythonic use of `any()` in initializing the variable (because something similar can be applied to the other two `uses_*`)